src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
```

Comment:
If you refer to this condition

> - cards container should have fixed paddings (see 1024px example), the bottom the same as the top

We need to change the shorthand padding because, in this variant of writing, top and bottom go first and then left and right

So the correct variant is `padding: 120px 0;`